Documentation/Settings.md

```diff
@@ -243,7 +249,7 @@ When powered by a battery, this adjusts the minimum voltage per cell before shut
 
 On device help text:
 
-Minimum allowed voltage per cell (3S: 3 - 3.7V | 4-6S: 2.4 - 3.7V)
+Minimum allowed volts per cell (3S: 3 - 3.7V | 4S-6S: 2.4 - 3.7V)
```

Comment:
Please leave it as it is.
There is nothing wrong about the sentence here!
If you want to add the 'S' after 4, you should also add the 'V' after 3 and after 2.4.
But if you were aware that this was the original state, then you would know that this change was made on purpose to save space/decrease the time to read the 'scrolling text'.

Reply:
"volts" is slang, it would be perferable to keep "voltage" if we can